tx_builder.go

```diff
@@ -143,6 +151,14 @@ func (tb *TxBuilder) MinCoinsForTxOut(txOut *TxOutput) Coin {
 func (tb *TxBuilder) calculateMinFee() Coin {
 	txBytes := tb.tx.Bytes()
 	txLength := uint64(len(txBytes))
+	// for each additional witnesses there will be an additional 100 bytes,
+	// (32 public key, 64 signature, 4 index/key in cbor)
+	txLength += uint64(tb.additionalWitnesses * 100)
```

Comment:
instead of doing this, we could just encode the tx using placeholder witnesses keys so we can have an accurate tx length

Reply:
what do you think? @safanaj

Reply:
hey @echovl do you mean adding some "fake" xprvkey with `.Sign` ? not sure how to do that and/or to get your idea.

Reply:
Hi @echovl I think I got your point, now in the calculateMinFee the additional witnesses are temporary added just to compute the length, the size of VKey and Signature are literal, those could be get from github.com/echovl/ed25519 but I didn't add that import just for 32 and 64 literals.